src/components/Summary.tsx

```diff
@@ -113,20 +115,34 @@ export default class Summary extends React.Component<SummaryProps, SummaryState>
                         return gpRef.type === "Practitioner" && gpRef.reference.includes(p.id);
                     });
                 });
-            notifyPractitionersSelector = <Autocomplete
+            const selectedPractitionersDisplay = currentSelection.length ?
+                currentSelection.map((p: IPractitioner, index) => {
+                    return <Chip
+                        key={`notify_p_display_${index}`}
+                        label={this.getPractitionerLabel(p)}
+                        variant="outlined"
+                        sx={{marginBottom: 0.5}}
```

Comment:
Is this needed? It looks basically identical when this line is removed. I tend to find that one-off sx properties just clutter the code and so I avoid them unless they're absolutely needed. (So if you think this one is important I'm okay with it 😊)

Reply:
thank you, Hannah.  I removed the one-off sx property.  I used the parent `Stack` element instead to resolve the spacing issue that I saw:
Before:
<img width="303" alt="Screen Shot 2023-04-16 at 1 18 06 PM" src="https://user-images.githubusercontent.com/12942714/232329796-1d568eb6-deb2-4090-95d0-802a80c0755f.png">
After:
<img width="329" alt="Screen Shot 2023-04-16 at 1 19 55 PM" src="https://user-images.githubusercontent.com/12942714/232329806-053ae9f4-681a-4ba2-8ea7-f34413780be8.png">